ctapipe/tools/display_dl1.py

```diff
@@ -173,15 +173,14 @@ class DisplayDL1Calib(Tool):
 
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
+        self.config.EventSource.input_url = get_dataset_path('gamma_test_large.simtel.gz')
```

Comment:
Why not just use the default value, rather than setting the config explicitly here?

Reply:
Where could I set the default? I don't want to set it globally for `EventSource` but just in this script

Reply:
Ah yes.  It's not possible without changing it in the EventSource. I had a similar problem in the DL1 tool for defining new defaults... This works, but it is a bit confusing as it looks like a configuration is being forced, even though it's overwritten later by the command-line.